assets/minecraft/lang/en_us.json

```diff
@@ -0,0 +1,810 @@
+  "subtitles.entity.panda.eat": "§aPanda eats",
+  "subtitles.entity.panda.step": "§aPanda steps",
+  "subtitles.entity.panda.cant_breed": "§aPanda bleats",
+  "subtitles.entity.panda.aggressive_ambient": "§cPanda huffs",
+  "subtitles.entity.panda.worried_ambient": "§aPanda whimpers",
+  "subtitles.entity.panda.hurt": "§aPanda hurts",
+  "subtitles.entity.panda.bite": "§aPanda bites",
+  "subtitles.entity.painting.break": "§aPainting breaks",
+  "subtitles.entity.painting.place": "§aPainting placed",
+  "subtitles.entity.phantom.ambient": "Phantom screeches",
+  "subtitles.entity.phantom.bite": "Phantom bites",
+  "subtitles.entity.phantom.death": "Phantom dies",
+  "subtitles.entity.phantom.flap": "Phantom flaps",
+  "subtitles.entity.phantom.hurt": "Phantom hurts",
+  "subtitles.entity.phantom.swoop": "Phantom swoops",
+  "subtitles.entity.pig.ambient": "§aPig oinks",
+  "subtitles.entity.pig.death": "§aPig dies",
+  "subtitles.entity.pig.hurt": "§aPig hurts",
+  "subtitles.entity.pig.saddle": "§aSaddle equips",
+  "subtitles.entity.pillager.ambient": "Pillager murmurs",
+  "subtitles.entity.pillager.celebrate": "Pillager cheers",
+  "subtitles.entity.pillager.death": "Pillager dies",
+  "subtitles.entity.pillager.hurt": "Pillager hurts",
+  "subtitles.entity.player.burp": "§dBurp",
+  "subtitles.entity.player.death": "§cPlayer dies",
+  "subtitles.entity.player.hurt": "§cPlayer hurts",
+  "subtitles.entity.player.levelup": "§bPlayer dings",
+  "subtitles.entity.polar_bear.ambient": "§aPolar Bear groans",
+  "subtitles.entity.polar_bear.ambient_baby": "§aPolar Bear hums",
+  "subtitles.entity.polar_bear.death": "§aPolar Bear dies",
+  "subtitles.entity.polar_bear.hurt": "§aPolar Bear hurts",
+  "subtitles.entity.polar_bear.warning": "§aPolar Bear roars",
+  "subtitles.entity.potion.splash": "§bBottle smashes",
+  "subtitles.entity.potion.throw": "§bBottle thrown",
+  "subtitles.entity.puffer_fish.blow_out": "§aPufferfish deflates",
+  "subtitles.entity.puffer_fish.blow_up": "§aPufferfish inflates",
+  "subtitles.entity.puffer_fish.death": "§aPufferfish dies",
+  "subtitles.entity.puffer_fish.flop": "§aPufferfish flops",
+  "subtitles.entity.puffer_fish.hurt": "§aPufferfish hurts",
+  "subtitles.entity.puffer_fish.sting": "§cPufferfish stings",
+  "subtitles.entity.rabbit.ambient": "§eFuwagi sniffs",
+  "subtitles.entity.rabbit.attack": "§eFuwagi hurts",
+  "subtitles.entity.rabbit.death": "§cFuwagi dies §f",
+  "subtitles.entity.rabbit.hurt": "§eFuwagi hurts",
+  "subtitles.entity.rabbit.jump": "§eFuwagi hops",
+  "subtitles.entity.salmon.flop": "§aSalmon flops",
+  "subtitles.entity.salmon.death": "§aSalmon dies",
+  "subtitles.entity.salmon.hurt": "§aSalmon hurts",
+  "subtitles.entity.sheep.ambient": "§aSheep baahs",
+  "subtitles.entity.sheep.death": "§aSheep dies",
+  "subtitles.entity.sheep.hurt": "§aSheep hurts",
+  "subtitles.entity.shulker.ambient": "Shulker lurks",
+  "subtitles.entity.shulker.close": "Shulker closes",
```

Comment:
Seems like a big section of vanilla sounds that aren't being changed. Are you planning to do something with these? If they won't be changed, I'd rather remove them from the file.

Reply:
i'll change all of the rest of the mob sounds in case there is an event that needs those mobs.